### PR TITLE
Vehicle route

### LINKED ISSUE #79: Adding vehicles route that provides vehicles data
**Describe the feature**
Provide vehicle data like vehicle name, manufacturer, type, colour and fuel.

**Describe the solution you'd like** :zap:
Add a new vehicle route.

**Are you working on this feature? (Yes)**

### PR DESCRIPTION
**Pull request by ritik307**
# Related Issue
- closes #79 

# Proposed Changes
- Added `vehicledata.json` in `data` folder
- Added `vehicles.ts` in `route` folder
- Updated `app.ts` with **vehicles** route

# Checklist
- [x] Tests
- [x] Translations

### CODE REVIEW

**Comment by ritik307** [comment]
@ShauryaAg  @paramsiddharth plz review

File: data/vehicledata.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "name": [
```
Comment (ShauryaAg) [inline data/vehicledata.json:2]: If you'd take a look at the previously added data. The schema is 

```
{
    "data": {
         ...data
    }
}

```

Please follow the same convention.

File: data/vehicledata.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    "color": ["black", "white", "green", "yellow", "blue", "red", "lime", "olive", "indigo"],
+    "vin": [
+        "U1MTJ97SUBEN55586",
+        "LK54TUF2ATVX14624",
+        "HJ2UNZV1L1SD11839",
+        "7J3HVRVMUXJT75742",
+        "2L2HM22394YH61393"
+    ],
+    "vrm": ["BX22GHM", "BC81XXS", "DI03UGZ", "MK89INV", "TE19HGU", "XR58HRL", "BS02LLL"]
+}
```
Comment (ShauryaAg) [inline data/vehicledata.json:42]: Also, I was expecting data to be more like:

```
{
   "data": [
    {
      "name": "Swift",
      "manufacturer": "Maruti",
      "type": "hatchback",
      "fuel": "petrol",
      ...
    },
    {
     ...
    }
  ]
}
```

**Review by ShauryaAg — CHANGES_REQUESTED** [review]

**Comment by ritik307** [comment]
@ShauryaAg Done! Plz the changes review

**Review by ShauryaAg — APPROVED** [review]

**Comment by ritik307** [comment]
@ShauryaAg  Done!

**Pull request merged**